packages/common/src/axios/create-instance.ts

```diff
@@ -88,5 +90,12 @@ export function createAxiosInstance({
     )(config)
   })
 
+  instance.interceptors.response.use(
+    async response => response,
+    async error => {
+      throw new SellingPartnerApiError(error)
```

Comment:
Can these only be `AxiosError` instances?

Reply:
It's already an `AxiosError`, but we want to do a custom error